internal/install/stack_version.go

```diff
@@ -6,5 +6,5 @@ package install
 
 const (
 	// DefaultStackVersion is the default version of the stack
-	DefaultStackVersion = "8.2.0"
+	DefaultStackVersion = "8.3.2"
```

Comment:
Should the Makefile target ([test-stack-command-8x](https://github.com/elastic/elastic-package/blob/main/Makefile#L64)) be updated to use that snapshot version ?

Reply:
Actually, the `8.3.0-SNAPSHOT` will pick up always latest `8.3.x`. In this case, it would the next patch release after 8.3.2, so we're safe :)

Reply:
Ok! 👍